spec/classes/repo_spec.rb

```diff
@@ -6,10 +6,10 @@
       let(:facts) { facts }
       if facts[:osfamily] == 'RedHat'
```

Comment:
@ghoneycutt Amazon Linux 2 is systemd based so needs the EL7 repos.  The fact sensu-go bootstrap only does EL6 is a bug in my opinion and opened here: https://github.com/sensu/sensu-go/issues/1791

Reply:
@ghoneycutt Uses big blocks that define facts could conflict and possibly break rspec-puppet-facts which looks up fact values based on what's supported in metadata.json.  Also mimics what values would be output by facter based on version so we're testing against Puppet 5 facts.  We could use the `os` value to build a mapping but that would get very verbose:

```
case os
when 'redhat-7-x86_64'
  something = something
when 'amazonlinux-2018-x86_64'
  something = something
else
  something = something
```

This would still rely on data from rspec-puppet-facts.